suite-native/state/package.json

```
@@ -38,7 +38,7 @@
         "react": "18.2.0",
         "react-native": "0.76.1",
         "react-redux": "8.0.7",
-        "redux-devtools-expo-dev-plugin": "^0.1.0",
+        "redux-devtools-expo-dev-plugin": "^1.0.0",
```

I'd go with `1.0.0`:
```suggestion
        "redux-devtools-expo-dev-plugin": "^1.0.0",
```